util/util.go

```diff
@@ -0,0 +1,63 @@
+package util
+
+import (
+	"fmt"
+	"regexp"
+)
+
+var fullCommandRegexp = regexp.MustCompile(`^([a-zA-Z0-9-_]+)\/([a-zA-Z0-9-_]+)@([a-z0-9-~\*\^\.]+)$`)
```

Comment:
I think command name and namespace should not start with `-` and `_` but [data-schema](https://github.com/screwdriver-cd/data-schema/blob/1375302b10cce8298d55d1b4a0a2e44ed80eb0a8/config/regex.js#L9-L11) is also same with this line...

Reply:
When I see the [data-schema regex.js](https://github.com/screwdriver-cd/data-schema/blob/master/config/regex.js), JOB_NAME and STEP_NAME are also allow to start with _, -. Thus I think now we do not need to have strict rule for command name and namespace.

When it becomes necessary, we should open PR for regex.

Reply:
🙆